db/seeds.rb

```diff
@@ -354,7 +378,7 @@
 Deal.new(id: '1009', title: '18 Chefs SG50 Set Meal 1 For 1', redeemable: true, multiple_use: true, type_of_deal: 'Freebies',
          description: 'Open Heart Surgery Fried Rice (for 4), Black Beauty, Peach in a Jar, Homemade Ice Lemon Tea x4 buy 1 get 1 free!',
          start_date: start_date, expiry_date: end_date, t_c: 'Not valid for Eighteen Chefs employees & their families. Dine in only',
-         pushed: true, merchant_id: 1002, active: true, num_of_redeems: 0, created_at: created_at).save(validate: false)
+         pushed: true, merchant_id: 1002, active: true, num_of_redeems: 0, created_at: created_at, push_date: activate_date).save(validate: false)
```

Comment:
@jkcheong92 noted, initially i left out activate_date as we will not be going into this accounts for presentation. Will add it in though